src/Email/Notifications/DonationProcessingReceipt.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Give\Email\Notifications;
+
+use Give\Donations\Models\Donation;
+use Give\Donations\ValueObjects\DonationStatus;
+use Give_Email_Notification;
+use Give_Payment;
+
+/**
+ * @unreleased
+ */
+class DonationProcessingReceipt extends Give_Email_Notification
+{
+    /* @var Give_Payment $payment */
+    public $payment;
```

Comment:
@glaubersilva you can remove this property too since its not used

Reply:
Resolved in 8cab5f4989eecfd72b96e15e1433cee7c11abef7